### PR TITLE
Applicants => aded list of users, aded filters for that

### PR DESCRIPTION
**Pull request by aghermanHP**
aded list of users, aded filters for that, aded image to activity, aded filter for activity, aded pagination for users invite list.
@nlaptedulce  please verify

### CODE REVIEW

File: src/assets/styles/_general.scss [hunk 1]
```diff
@@ -49,6 +49,11 @@ body {
     margin-left: 5px;
   }
 
+  .btn-delete {
+    background-image: linear-gradient(to right, #ffda00 0%, #f54a00 50%, #ff0000 100%);
+    color: #800000;
```
Comment (nlaptedulce) [inline src/assets/styles/_general.scss]: you can add this color as a variable

File: src/assets/styles/_general.scss [hunk 2]
```diff
@@ -74,16 +79,19 @@ body {
     cursor: pointer;
   }
 
+  .page-link{
```
Comment (nlaptedulce) [inline src/assets/styles/_general.scss]: remove this empty class

File: src/assets/styles/_general.scss [hunk 3]
```diff
@@ -74,16 +79,19 @@ body {
...
+
   .line {
     padding: 1px;
     background-image: var(--lineGradient);
   }
 
   .page-item.active .page-link {
-    background: var(--orangeGradient);
-    border-color: var(--orange);
+    background: #5a6268;
```
Comment (nlaptedulce) [inline src/assets/styles/_general.scss:83]: you can add this color as a variable

File: src/components/activity/Technologies.vue [hunk 4]
```diff
@@ -19,7 +22,7 @@
       prop: 'technologies',
       event: 'changeTechnologiesList'
     },
-    props: { technologies: Array },
+    props: { technologies: Array},
```
Comment (nlaptedulce) [inline src/components/activity/Technologies.vue]: after Array a space

File: src/components/activity/UsersFiltredList.vue [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+        <p v-else-if="row.item.seniority === 1">Middle</p>
+        <p v-else-if="row.item.seniority === 2">Senior</p>
+      </template>
+      
+      <template slot="row-details" slot-scope="row">
+        <b-card class="border-info" @click="row.toggleDetails">
+          <div class="row col-md-12">
+            Technologies:
+            <div v-for="(myTechnology, comas) in row.item.technologies" :key="myTechnology">
+              <hdiv class="row ml-3">{{ myTechnology.name }}<p v-if="comas+1 < row.item.technologies.length">,  &nbsp;</p></hdiv>
```
Comment (nlaptedulce) [inline src/components/activity/UsersFiltredList.vue]: hdiv?

File: src/components/activity/UsersFiltredList.vue [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+          results: [
+            { technologies: [] },
+            { stars: null }
+          ]
+        },
+        filter: '/users',
+        options:[
+          {value: '/users', text:'All users are selected'},
+          {value:  '/users?filter[activityRole][]=0', text: 'Invited users'},
+          {value: '/users?filter[activityRole][]=1', text: 'Applied users'},
```
Comment (nlaptedulce) [inline src/components/activity/UsersFiltredList.vue:110]: you can write this as constant in an other file

Comment (aghermanHP) [reply src/components/activity/UsersFiltredList.vue:110]: i think i don't need this in any other components :)

File: src/assets/styles/_general.scss (hunk 1 above)
Comment (aghermanHP) [reply src/assets/styles/_general.scss]: i deleted that class

File: src/assets/styles/_general.scss (hunk 2 above)
Comment (aghermanHP) [reply src/assets/styles/_general.scss]: ok

File: src/assets/styles/_general.scss (hunk 3 above)
Comment (aghermanHP) [reply src/assets/styles/_general.scss:83]: that are variables allredy

File: src/components/activity/Technologies.vue (hunk 4 above)
Comment (aghermanHP) [reply src/components/activity/Technologies.vue]: ok

File: src/components/activity/UsersFiltredList.vue (hunk 5 above)
Comment (aghermanHP) [reply src/components/activity/UsersFiltredList.vue]: just div sorry

**Pull request merged**